src/main/java/dk/creditoro/client/core/ViewHandler.java

```diff
@@ -65,6 +65,28 @@ public boolean openView(Views viewToOpen) {
         return true;
     }
 
+    public boolean openView(Views viewToOpen, String param) {
+        FXMLLoader loader = new FXMLLoader();
+        Scene scene;
+        try {
+            scene = getScene(viewToOpen, loader);
+        } catch (IOException e) {
+            LOGGER.info(String.format("Coudn't find FXML file with name: %s", viewToOpen));
+            return false;
+        }
+        if (viewToOpen == Views.BROWSE_CHANNELS) {
```

Comment:
I'm not sure I understand the use of this if-else statement, could you please explain?

Reply:
if searching in channels. then search this way. if searching on productions search this other way

Reply:
Oh right, it's the one you overloaded. I forgot, my bad, no worries here.